controllers/comparators/serviceaccount_comparator.go

```diff
@@ -0,0 +1,11 @@
+package comparators
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetServiceAccountComparator() ResourceComparator {
+	return func(deployed client.Object, requested client.Object) bool {
+		return true
```

Comment:
Are you planning to add some logic here?

Reply:
no.. for now we just create a blank ServiceAccount object and didn't set any Spec field. Thats why I directly return true here